src/_packagedcode/pypi.py

```diff
@@ -22,15 +22,15 @@
 import zipfile
 
 import dparse2
-import packaging
```

Comment:
This pypi.py modules comes originally from ScanCode toolkit... Can you check how the modifications were done there and have the same modifications?